docs/howto/manage-web-app-charms/use-web-app-charm.rst

```diff
@@ -28,3 +28,65 @@ script will run on every unit. The script is assumed to be idempotent (in other
 can be run multiple times) and that it can be run on multiple units simultaneously
 without issue. Handling multiple migration scripts that run concurrently
 can be achieved by, for example, locking any tables during the migration.
+
+Use observability
+-----------------
+
+First, :ref:`integrate your web app with the Canonical Observability
+Stack <integrate_web_app_cos>`.
+
+Get Grafana login details
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+View the endpoints by running:
+
+.. code-block::
+
+    juju show-unit catalogue/0 | grep url
+    juju run grafana/leader get-admin-password
+
+These commands output URLs and the default admin password. Look for the URL
+with the Grafana postfix with the syntax
+``http://<IP_ADDRESS>/<JUJU_MODEL_NAME>-grafana``. Here, ``JUJU_MODEL_NAME``
+is the name of the Juju model on which you deployed your web app.
+
+View Grafana dashboard
+~~~~~~~~~~~~~~~~~~~~~~
+
+To view the dashboard, append the ``/dashboards``
+postfix to the URL and log in using the admin password to view the dashboards
+overview page.
+
+To view the specific dashboard for your web app, click :guilabel:`General` and
+then on :guilabel:`WebApp Operator`, where "WebApp" is a stand-in for the
+framework of your web app.
```

Comment:
A minor point, but I don't find that the look/feel of `guilabel` is in line with the small bits we have in the style guide about [documenting UI](https://docs.ubuntu.com/styleguide/en/#interacting-with-ui-elements). I suppose there's nothing we can do about the theme here, but I would personally be fine if we used simple bold.